processor/integrations/integrations.go

```diff
@@ -19,16 +20,31 @@ var (
 	destTransformURL, userTransformURL string
 	customDestination                  []string
 	whSchemaVersion                    string
+	postParamentersTFields               []string
```

Comment:
Typo? postParamentersTFields

Reply:
I used it for the lack of better name.

Reply:
I meant PostParamenters* has an extra "n"
It should PostParameters*